terraform/fullnode/digital_ocean_localstorage/kubernetes.tf

```diff
@@ -0,0 +1,106 @@
+provider "kubernetes" {
+  host                   = "${digitalocean_kubernetes_cluster.aptos.endpoint}"
+  cluster_ca_certificate = base64decode(digitalocean_kubernetes_cluster.aptos.kube_config[0].cluster_ca_certificate)
+  token                  = digitalocean_kubernetes_cluster.aptos.kube_config[0].token
+}
+
+resource "kubernetes_namespace" "aptos" {
+  metadata {
+    name = var.k8s_namespace
+  }
+}
+
+resource "kubernetes_storage_class" "local" {
+  storage_provisioner = "kubernetes.io/no-provisioner"
+  volume_binding_mode = "WaitForFirstConsumer"
+
+  metadata {
+    name = "local-storage"
+    annotations = {
+      "storageclass.kubernetes.io/is-default-class" = "true"
+    }
+  }
+}
+
+resource "kubernetes_persistent_volume" "pv" {
```

Comment:
I think if you just specify the storage type, you don't need to create this, and it should be taken care of by the helm installation.